extensions/stackdriver/stackdriver.cc

```diff
@@ -186,6 +186,30 @@ absl::Duration getMetricExpiryDuration(
          absl::Nanoseconds(duration.nanos());
 }
 
+std::vector<std::string_view> getDroppedMetrics(
+    const stackdriver::config::v1alpha1::PluginConfig& config) {
+  std::vector<std::string_view> dropped_metrics;
```

Comment:
Should we make this a map?

Reply:
i don't think it needs to be a map. there will never be many in the list in any case.